libcudacxx/include/cuda/std/detail/libcxx/include/span

```diff
@@ -330,6 +331,13 @@ public:
       : __data_{nullptr}
   {}
 
+  template <class _Up = element_type, enable_if_t<_CCCL_TRAIT(is_const, _Up), int> = 0>
```

Comment:
This should follow the other constructors and use our `_LIBCUDACXX_TEMPLATE` macro that will be changed to `_CCCL_TEMPLATE` soon

```suggestion
  _CCCL_TEMPLATE(class _Tp2 = _Tp)
  _CCCL_REQUIRES(_CCCL_TRAIT(is_const, _Tp2)
```